app/models/mixins/authentication_mixin.rb

```diff
@@ -191,9 +191,36 @@ def authentication_type(type)
     end
   end
 
+  MAX_ATTEMPTS = 6
+  # The default for the schedule is every 1.hour now.
+  #   6 will gives us:
+  #   A failure now and retries in 2, 4, 8, and 16 minutes, for a total of 30 minutes.
+  #   We'll wait another 30 minutes, minus the time it takes to queue and perform the checks
+  #   before the schedule fires again.
+  def authentication_check_retry_deliver_on(attempt)
+    # Existing callers who pass no attempt will have no delay.
+    case attempt
+    when nil, 0, ->(a) { a >= MAX_ATTEMPTS }
```

Comment:
Hold off on merging, I think there's a bug in this code.  I need to verify it doesn't queue up an "immediate" retry on that `a >= MAX_ATTEMPTS` situation.